CONTRIBUTING.md

```diff
@@ -2,7 +2,28 @@
 
 Enhancements and fixes are always welcome. Feel free to contribute to any of the Issues not already assigned to another person.
 
-* `pip install -r dev_requirements.txt`
-* `cd tests`
-* `pytest`
-* New code should have unit tests coverage
+1. Install [Python 3](https://www.python.org/downloads/) and pip
+1. Install [dotent 5.0 runtime](https://dotnet.microsoft.com/download/dotnet/5.0/runtime)
+1. Clone this repository
+1. In the repository modify the `configuration.local.json`
```

Comment:
in the repo it's got `configuration.example.json`, presumably you mean to rename it and modify it?